packages/evm/src/evm.ts

```diff
@@ -663,6 +676,7 @@ export class EVM implements EVMInterface {
       )
       gasLimit -= createCompleteAccessGas
       if (gasLimit < BIGINT_0) {
+        message.accessWitness?.clearCache()
         if (this.DEBUG) {
           debug(
             `ContractCreateComplete access gas (${createCompleteAccessGas}) caused OOG (-> ${gasLimit})`,
```

Comment:
Should we also add a corresponding flushCaches in the else statement for contractCreateComplete?